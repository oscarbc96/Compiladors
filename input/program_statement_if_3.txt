calc off

a := 20.0

if (a > 20) then
  123
  123
  123
elsif (a < 20) then
  123
  123
  123
elsif (a = 20) then
  123
  123
  123
else
  123
  123
  123
fi
