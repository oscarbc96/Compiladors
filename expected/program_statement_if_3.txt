1: $t01 := 20.000000
2: a := $t01
3: IF a GTF 20 GOTO 5
4: GOTO 12
5: PARAM 123
6: CALL PUTI,1
7: PARAM 123
8: CALL PUTI,1
9: PARAM 123
10: CALL PUTI,1
11: GOTO 36
12: IF a LTF 20 GOTO 14
13: GOTO 21
14: PARAM 123
15: CALL PUTI,1
16: PARAM 123
17: CALL PUTI,1
18: PARAM 123
19: CALL PUTI,1
20: GOTO 36
21: IF a EQ 20 GOTO 23
22: GOTO 30
23: PARAM 123
24: CALL PUTI,1
25: PARAM 123
26: CALL PUTI,1
27: PARAM 123
28: CALL PUTI,1
29: GOTO 36
30: PARAM 123
31: CALL PUTI,1
32: PARAM 123
33: CALL PUTI,1
34: PARAM 123
35: CALL PUTI,1
36: HALT
